ENIGMAsystem/SHELL/Platforms/General/PFmain.cpp

```diff
@@ -47,6 +48,8 @@ void set_program_args(int argc, char** argv) {
 }
 
 int enigma_main(int argc, char** argv) {
+  enigma::initGame = false;
```

Comment:
Why initialize this here instead of the line where it's defined? Also, have you considered how this variable interacts with `game_restart`?

Reply:
it does not get rest to false when game_restart() is called. This is intentional. We already have a gamestart variable in roomsystem.cpp; this is for an entirely different purpose, and the purpose is to apply windowing-related global game settings once and only once for each instance of the game running.

Reply:
Actually, you don't need `initGame` at all, `gamestart` is already a paramter, you can use it and remove `initGame` now.
https://github.com/enigma-dev/enigma-dev/blob/d606db0007d3b9801e6290bd4271b1d24c7eb142/ENIGMAsystem/SHELL/Universal_System/roomsystem.cpp#L190